enterprise-suite/gotests/util/util.go

```diff
@@ -124,14 +124,20 @@ func (cb *CmdBuilder) Run() (int, error) {
 		panic("unable to get command stderr pipe")
 	}
 
+	// Always capture stdout and stderr to these string builders so that
+	// we can print them out in case of errors
+	var errStdoutBuilder strings.Builder
+	var errStderrBuilder strings.Builder
```

Comment:
Can you use the `GinkgoWriter`? It will auto print out only if a test fails.

Reply:
Switched to GinkgoWriter here